### PR TITLE
[WINSPOOL] printers.c: Demote 3 ERR() to TRACE(), Fix 1 copypasta

### PR DESCRIPTION
**Pull request by SergeGautherie**

### CODE REVIEW

**Comment by learn-more** [comment]
Why?

**Comment by SergeGautherie** [comment]
> Why?

Why what? Because these traces should not be mis-identified as errors?

File: win32ss/printing/base/winspool/printers.c [hunk 1]
```diff
@@ -1050,7 +1050,7 @@ GetPrinterDriverW(HANDLE hPrinter, LPWSTR pEnvironment, DWORD Level, LPBYTE pDri
     DWORD dwErrorCode;
     PSPOOLER_HANDLE pHandle = (PSPOOLER_HANDLE)hPrinter;
 
-    ERR("GetPrinterDriverW(%p, %S, %lu, %p, %lu, %p)\n", hPrinter, pEnvironment, Level, pDriverInfo, cbBuf, pcbNeeded);
+    TRACE("GetPrinterDriverW(%p, %S, %lu, %p, %lu, %p)\n", hPrinter, pEnvironment, Level, pDriverInfo, cbBuf, pcbNeeded);
```
Comment (Extravert-ir) [inline win32ss/printing/base/winspool/printers.c]: This one makes sense, other ones are not because they indicate use of `UNIMPLEMENTED` function

**Comment by SergeGautherie** [comment]
> `UNIMPLEMENTED`

I assumed this macro is enough by itself.
Nonetheless, if `ERR(); UNIMPLEMENTED;` pattern is wanted, then let's enforce it for the whole file?

File: win32ss/printing/base/winspool/printers.c (hunk 1 above)
Comment (learn-more) [reply win32ss/printing/base/winspool/printers.c]: Either way, it was just committed, and it is quite rude to follow commits immediately by another commit modifying stuff....

**Comment by jimtabor** [comment]
Hi,
It would not be an issue.
But, currently working on GDI support for printing. It's in its final cut. These stubs print function will be required to be implemented within a year. (Sounds like a nice project for someone......)

**Comment by ColinFinck** [comment]
@SergeGautherie: We don't really have a strict style to follow for unimplemented functions, so please refrain from changing any further `ERR` to `TRACE` for them. This will eventually be dealt with when a function is properly implemented.
Good catch on the `_RpcGetPrinterDriver` though. This was indeed a copy-paste mistake.

**Pull request merged**